src/Microsoft.DotNet.XHarness.iOS.Shared/TestImporter/Templates/Managed/Resources/src/common/TestRunner/Core/TcpTextWriter.cs

```diff
@@ -88,7 +91,27 @@ public TcpTextWriter(string hostName, int port)
 
             try
             {
-                client = new TcpClient(HostName, port);
+                if (isTunnel)
+                {
+                    server = new TcpListener(IPAddress.Any, Port);
```

Comment:
Does this need disposing or we just don't care and wait for the app to end completely?